gulpfile.js

```diff
@@ -2,12 +2,14 @@ var gulp = require('gulp'),
     browserify = require('browserify'),
     buffer = require('vinyl-buffer'),
     rename = require('gulp-rename'),
+    run = require('gulp-run'),
```

Comment:
@mprudhom: Is this used for anything? If not, can we drop it here and from `package.json`?